README.md

```diff
@@ -3,65 +3,86 @@ Web3j Open API
+abstracting the coding layer. These interactions can be done using :
+- plain HTTP requests
+- via the Swagger-UI, which is generated with every project
+- A client application utilizing the `webj3-openapi-client` client implementation
 
-It builds on top of Web3j's smart contract wrappers to provide a fully typesafe integration from the client through to the blockchain.
+The workflow can be summed in the following steps:
+- Writing a solidity smart contract
+- Generating the corresponding **OpenAPI** project using **Web3j-OpenAPI**
+- Running the generated project as a standalone server
+- Sending HTTP requests using `Swagger-UI`, client application or `Curl` request.
 
-## Modules
 
- * openapi: JAX-RS, Web3j Core (only interfaces), Swagger annotations
- * core: openapi, Jersey Core, Swager UI
- * server: core, Jersey Server, Jetty, integration tests
- * codegen: Mustache, Kotlinpoet?, openapi (Based on CorDappGenerator for Gradle project)
- * client: core, Jersey Client
- * console: Command-line tools command `epirus-openapi`, sub-commands `generate`, `run`
- * helloworld:
-   * openapi
-   * server: web3j-openapi-server (org.web3j.openapi.server.MainKt or generated main)
-
-## Use cases
+# Getting started with Web3j-OpenAPI
 
+To generate an OpenAPI project using the [Web3j-OpenAPI](https://github.com/web3j/web3j-openapi) generator, you need to have the [Epirus-CLI](https://docs.epirus.io/quickstart) installed on your machine (Note - the Epirus CLI has replaced the Web3j CLI). 
+It’s easy to do (for Windows instructions head [here](https://docs.epirus.io/quickstart/#installation)):
+	
 ```ssh
-$ epirus generate openapi Greeter.sol -p com.helloworld -o helloworld
+$ curl -L get.epirus.io | sh
 ```
 
-Creates Gradle project with one module to run a server (with application plugin) and another with interfaces.
+### Create a Hello World project
+To create a base OpenAPI project using a `Hello World` contract, run the following :
+
 ```ssh
-$ cd /home/rachid/helloworld
-$ ./gradle run // Starts the server exposing Greeter.sol
-...
+$ epirus openapi new
 ```
 
-Also start the server with:
+You can also generate a `Web3j-OpenAPI` project using the [Web3j-OpenAPI-gradle-plugin](https://github.com/web3j/web3j-openapi-gradle-plugin).
+
+### Configure the project
+After having the generated project, you can either use the [Epirus-cli](https://docs.epirus.io/quickstart/#deployment) and easily run it.
+Or configure it. The simplest way is using environment variables :
+
 ```ssh
-$ ./gradle shadowJar // Create an executable JAR that starts the server
-$ java -jar helloworld-0.1.0-all.jar
-...
+$ export WEB3J_ENDPOINT=<link_to_your_Ethereum_node>
+$ export WEB3J_PRIVATE_KEY=<your_private_key>
+$ export WEB3J_OPENAPI_HOST=localhost
+$ export WEB3J_OPENAPI_PORT=9090
 ```
 
-Or using the CLI:
+### Run the project
+The project can be run in multiple ways. In this example, we will go for:
```

Comment:
If you aren't using the Epirus CLI, you may run the project using the following Gradle target: